lib/client-api/src/ClientApi.ts

```diff
@@ -396,10 +397,12 @@ Read more here: https://github.com/storybookjs/storybook/blob/master/MIGRATION.m
   };
 
   getStorybook = (): GetStorybookKind<TFramework>[] => {
-    const { stories } = this.storyStore.storyIndex;
+    const { entries } = this.storyStore.storyIndex;
 
     const kinds: Record<ComponentTitle, GetStorybookKind<TFramework>> = {};
-    Object.entries(stories).forEach(([storyId, { title, name, importPath }]) => {
+    Object.entries(entries).forEach(([storyId, { title, name, importPath, type }]) => {
+      if (type && type !== 'story') return;
```

Comment:
Should we normalize it when we load it into the store so we don't have to do these null checks all over the place?